src/main.rs

```diff
@@ -230,7 +230,10 @@ fn settings() -> io::Result<Settings> {
     // Determine the minimum age for images to be considered for deletion.
     let min_age = match matches.value_of(MIN_AGE_OPTION) {
         Some(value) => match parse(value) {
-            Ok(duration) => Some(duration),
+            Ok(duration) => {
+                debug!("min-age argument parsed as {:?}", duration);
```

Comment:
Could we do something like:

```rust
debug!("{:?} parsed as {:?}.", format!("--{:?}", MIN_AGE_OPTION).code_str(), duration);
```

Reply:
Dropped the first "?" I think is what you want.  I'm a bit new at rust, but that seems better

Reply:
Ah yes you are right! I think the `:` is unnecessary there too (so `{}` instead of `{:}`). Would you mind removing it? Overall the PR LGTM.

Reply:
done and did clippy check locally this time